Misc/NEWS.d/next/Core and Builtins/2020-01-30-14-36-31.bpo-39502.IJu0rl.rst

```diff
@@ -0,0 +1,2 @@
+Fix :func:`time.localtime` on 64-bit AIX  to support years before 1902 and after 2038
+Patch by M Felt
```

Comment:
Same here :-)

Reply:
Naughty periods - hiding like that. Applied permanent ink!